src/App.js

```diff
@@ -1,27 +1,12 @@
-import { DatePicker } from 'antd';
-
-import logo from './logo.svg';
 import './App.css';
+import SignUp from './Signup/SignUp';
```

Comment:
the path of the component doesn't make send. `/Signup/SignUp`

Reply:
It's correct.